pkg/dry/math.go

```diff
@@ -16,13 +16,19 @@ func Min(x, y int64) int64 {
 	return y
 }
 
-// CeilToMultiplier returns the integer greater or equal to x and multiplier m product.
+// Ceil returns integer greater or equal to x and denominator d division.
 // Works only with x >= 0 and m > 0. It returns 0 with other values.
```

Comment:
It looks like it should be `d > m` here too

```suggestion
// Works only with x >= 0 and d > 0. It returns 0 with other values.
```